assets/js/components/settings/SettingsStatuses.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * SettingsStatuses component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+export default function SettingsStatuses( { statuses } ) {
+	if ( ! statuses || statuses.length === 0 ) {
+		return null;
+	}
+
+	return (
+		<Fragment>
+			<div className="googlesitekit-settings-module__meta-items">
+				{ statuses.map( ( { label, status } ) => (
+					<div
+						key={ label }
+						className="googlesitekit-settings-module__meta-item"
+					>
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							{ label }
+						</h5>
+						<p className="googlesitekit-settings-module__meta-item-data">
+							{ status !== undefined && status }
+						</p>
+					</div>
+				) ) }
+			</div>
+		</Fragment>
```

Comment:
As per the above, there's no need for the `Fragment`.

```suggestion
		<div className="googlesitekit-settings-module__meta-items">
			{ statuses.map( ( { label, status } ) => (
				<div
					key={ label }
					className="googlesitekit-settings-module__meta-item"
				>
					<h5 className="googlesitekit-settings-module__meta-item-type">
						{ label }
					</h5>
					<p className="googlesitekit-settings-module__meta-item-data">
						{ status !== undefined && status }
					</p>
				</div>
			) ) }
		</div>
	);
```